art/attacks/inference/membership_inference/label_only_boundary_distance.py

```diff
@@ -196,6 +202,62 @@ def calibrate_distance_threshold(
 
         self.distance_threshold_tau = distance_threshold_tau
 
+    def calibrate_distance_threshold_unsupervised(self, top_t: int, num_samples: int, max_queries: int, **kwargs):
```

Comment:
Is it possible to provide default values for the params top_t, num_samples and max_queries?

Reply:
Of course! I would suggest top_t to be set to 50% (because in the paper the best attack performances were achieved between 30% and 80%). 
num_samples could be 100 for example? And max_queries 1, so it doesn't run forever. But these values are pretty much ad-hoc.

Reply:
That's fine. The user can always override them, but it's good to have the option (especially for non-experts) to run it without having to supply anything.